public/app/features/panel/panel_ctrl.ts

```diff
@@ -193,7 +193,7 @@ export class PanelCtrl {
       });
 
       menu.push({
-        text: 'Add to Panel List',
+        text: 'Copy',
         click: 'ctrl.addToPanelList()',
```

Comment:
Maybe rename `addToPanelList` to `copyPanel` now? 

Another nice to have would be a keyboard shortcut (maybe P+C). But this is not a blocker for now.